nns/nns_contract.go

```diff
@@ -276,27 +282,37 @@ func Register(name string, owner interop.Hash160, email string, refresh, retry,
 			panic("TLD already exists")
 		}
 		storage.Put(ctx, tldKey, 0)
-	} else {
-		if tldBytes == nil {
-			panic("TLD not found")
-		}
-		if parentExpired(ctx, 1, fragments) {
-			panic("one of the parent domains is not registered")
-		}
-		parentKey := getTokenKey([]byte(name[len(fragments[0])+1:]))
+		putNameStateWithKey(ctx, getTokenKey([]byte(name)), NameState{
+			Name: name,
+			// NNS expiration is in milliseconds
+			Expiration: int64(runtime.GetTime() + expire*1000),
+		})
```

Comment:
`ownerOf` method of such domain will return an empty hash. Is it correct from the user's side of view?

Reply:
fixed